### ISSUE TITLE
Accessibility: https://mybinder.readthedocs.io/en/latest/using/using.html#preparing-a-repository-for-binder

### ISSUE THREAD

**Issue by Josephsanya** [jupyterhub/outreachy#115]
### What page is this for?

https://mybinder.readthedocs.io/en/latest/using/using.html#preparing-a-repository-for-binder

### WAVE accessibility report

I'm Joseph Sanyaolu, an Outreachy applicant who worked on the website [mybinder.org](https://mybinder.readthedocs.io/en/latest/using/using.html#preparing-a-repository-for-binder)
### **BINDER.ORG WAVE ACCESSIBILITY RESULTS**
![Screenshot (4)](https://user-images.githubusercontent.com/57257090/197390801-98cd88a3-a798-411f-b9cc-7dfaa7d5fde1.png)
I evaluated the website using WAVE accessibility tool and the following is the result:
  
-- **1 Errors**
 --This is due to a missing alternative text to an image
 --FIX: an alternative text should be added to the image which will describe the meaning and content of the image using the alt attribute.

-- **3 Contrast Errors**
 --This is due to very low contrast between text and background colors. 
 --FIX: We can increase the contrast between the foreground which is the text color and the background color.

-- **3 Yellow Alerts (2 Suspicious alternative text & 1 redundant link)**
 --Suspicious alternative text is caused by insufficient text description or a text containing extraneous information
 --FIX: Ensure that the alternative text for the image or image input provides a succinct, yet equivalent alternative to the content and function of the image
 --Redundant link is caused by adjacent links going to the same URL
 --FIX: Combine the redundant links into one link and remove any redundant text or alternative text.

We got 3 features, which means we have well implemented features, 15 structural elements which shows that our website is well structed and 7 ARIAs.

### The HTML element to be changed

```html
<!-- Errors -->
<img src="https://server.ethicalads.io/proxy/view/3070/83ec89cb-4b6f-4f63-9175-e156a2f716ee/" class="ea-pixel">
<img src="https://server.ethicalads.io/proxy/viewtime/3070/83ec89cb-4b6f-4f63-9175-e156a2f716ee/?view_time=7" class="ea-pixel">

<!-- Contrast Errors -->
<a class="btn btn-secondary support-button" 
href="https://mybinder.readthedocs.io/en/latest/about/support.html" 
role="button" style="color: rgb(255, 255, 255); background-color: rgb(230, 103, 131);">
    <i class="fas fa-heart"></i> 
    Support Binder
</a>

<!-- Suspicious Alternative texts -->
<img src="../_static/logo.png" class="logo__image only-light" alt="Logo image">
<img src="../_static/logo.png" class="logo__image only-dark" alt="Logo image">

<!-- Redundant link -->
<a href="https://server.ethicalads.io/proxy/click/3071/0c5a224a-5651-4e61-a6e5-91d765763a9e/"
rel="nofollow noopener" target="_blank">
    <strong>
    Read the Docs for Business:
    </strong>
    <span>
    Automated docs deployment of private & public repos.
    </span>
    <strong>
    Get started today.
    </strong>
    </a>
```

### Your proposed HTML (or CSS), after change

```html
<!-- Errors -->
<!-- Added alt attribute -->
<img src="https://server.ethicalads.io/proxy/view/3070/83ec89cb-4b6f-4f63-9175-e156a2f716ee/" class="ea-pixel" alt="">
<img src="https://server.ethicalads.io/proxy/viewtime/3070/83ec89cb-4b6f-4f63-9175-e156a2f716ee/?view_time=7" class="ea-pixel" alt="">

<!-- Contrast Errors -->
<!-- Modified background-color -->
<a class="btn btn-secondary support-button" 
href="https://mybinder.readthedocs.io/en/latest/about/support.html" 
role="button" style="color: rgb(255, 255, 255); background-color: rgb(97, 15, 33);">
    <i class="fas fa-heart"></i> 
    Support Binder
</a>

<!-- Suspicious Alternative texts -->
<!-- Changed alt text to match img -->
<img src="../_static/logo.png" class="logo__image only-light" alt="Logo image light">
<img src="../_static/logo.png" class="logo__image only-dark" alt="Logo image dark">

<!-- Redundant link -->
<a href="https://server.ethicalads.io/proxy/click/3071/0c5a224a-5651-4e61-a6e5-91d765763a9e/"
rel="nofollow noopener" target="_blank">
    <strong>
    Read the Docs for Business:
    </strong>
    <span>
    Automated docs deployment of private & public repos.
    </span>
    <strong>
    Get started today.
    </strong>
    </a>
```

**Comment by minrk** [comment]
Thanks for the suggestions!

**Comment by Josephsanya** [comment]
I'm Glad
You're welcome!

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
